src/js/providers/html5.js

```diff
@@ -282,6 +282,24 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
         return duration;
     };
 
+    _this.getSeekRange = function() {
+        const seekRange = {
+            start: 0,
+            end: 0
+        };
+
+        const seekable = this.video.seekable;
+
+        if (seekable.length) {
+            const maxSeekableEnd = Math.max(seekable.end(0), seekable.end(seekable.length - 1));
+
+            seekRange.end = maxSeekableEnd;
+            seekRange.start = Math.max(0, seekRange.end - this.video.duration);
```

Comment:
Why not use `seekable.start(0)`?

or 

`Math.min(seekable.start(0), seekable.start(seekable.length - 1));`